libs/akita-ng-effects/src/lib/module-manager.service.ts

```diff
@@ -0,0 +1,46 @@
+import { Injectable, OnDestroy, Type } from '@angular/core';
+import { Subject } from 'rxjs';
+import { takeUntil } from 'rxjs/operators';
+import { Actions } from './actions';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ModuleManager implements OnDestroy {
+  effectInstanceSources: Type<any>[] = [];
+  destroyEffects$ = new Subject();
+
+  constructor(private actions$: Actions) {}
+
+  addEffectInstance(effectInstance) {
```

Comment:
Please add types to any argument in this file

Reply:
Types added.